### PR TITLE
fix skull position in help screen for `gamemode == commercial`

### LINKED ISSUE #826: Skull graphic is misplaced in F1 help screen for -complevel 2.
I know that this may come off as an incredibly minuscule nitpick, but I couldn't help pointing out an odd inconsistency I noticed in the help screens involving the skull in the corner.

In the F1 help screen, the skull at the bottom right corner is supposed to be just above the text. Here's the screen taken in Chocolate Doom.
help_chocolate

However in Woof, the skull is lower than it's supposed to be. It's actually using the skull position that found in the Doom 1 help screens.
help_woof

This only occurs in Doom 2 / -complevel 2. In Final Doom / -complevel 4, the skull is in the correct position on the help screen.

### PR DESCRIPTION
**Pull request by rfomin**
Fix #826

### CODE REVIEW

File: src/m_menu.c [hunk 1]
```diff
@@ -6904,6 +6904,7 @@ void M_Init(void)
       ReadDef1.routine = M_DrawReadThis1;
       ReadDef1.x = 330;
       ReadDef1.y = 165;
+      HelpDef.y = 165;
```
Comment (fabiangreffrath) [inline src/m_menu.c:6907]: Does this also look alright with the dynamic help screen?

Comment (rfomin) [reply src/m_menu.c:6907]: Yes, we have redesigned the dynamic help screen:

woof0001

Comment (fabiangreffrath) [reply src/m_menu.c:6907]: I know! 😁 I just couldn't imagine if the moved skull would still fit well.

**Comment by fabiangreffrath** [comment]
But why was this different between complevels 2 and 4?

**Comment by rfomin** [comment]
> But why was this different between complevels 2 and 4?

For me it's the same on all complevels. It's just that HelpDef has coordinates from Doom 1.

**Pull request merged**